### PR TITLE
update some page

### PR DESCRIPTION
**Pull request by pitb2022**
update Recently Written Card background
update Library background color and padding top
add title & fetch own logbook
run `npm run format`

### CODE REVIEW

File: src/views/Library/index.tsx [hunk 1]
```diff
@@ -45,6 +45,18 @@ const Library: React.FC = () => {
       </section>
 
       <BookList />
+      <style jsx global>
```
Comment (robertu7) [inline src/views/Library/index.tsx:48]: why not CSS modules?

File: src/views/Library/styles.module.css [hunk 2]
```diff
@@ -1,6 +1,8 @@
 .header {
   @mixin flex-center-space-between;
 
+  padding-top: 80px;
```
Comment (robertu7) [inline src/views/Library/styles.module.css:4]: rem instead of px?

File: src/views/Library/index.tsx (hunk 1 above)
Comment (pitb2022) [reply src/views/Library/index.tsx:48]: only this page background color is white, other page background is grey-lighter

**Comment by tx0c** [comment]
19 commits behind HEAD and 3 commits ahead, better do a rebase now?

File: src/components/LogbookCard/Footer/index.tsx [hunk 3]
```diff
@@ -27,21 +26,21 @@ export const Footer: React.FC<FooterProps> = ({
...
   }
 
-  let borderRadius = '1rem'
-  let width = '3.25rem'
-  let height = '1.75rem'
-  let hashWidth = '5.1875rem'
+  let borderRadius = "1rem";
+  let width = "3.25rem";
+  let height = "1.75rem";
+  let hashWidth = "5.1875rem";
```
Comment (tx0c) [inline src/components/LogbookCard/Footer/index.tsx:38]: better use `const` for not-changing values?

**Pull request merged**